plugins/extractors/oracle/README.md

```diff
@@ -0,0 +1,45 @@
+# Oracle
+
+## Usage
+
+```yaml
+source:
+  type: oracle
+  config:
+    host: localhost:1521
```

Comment:
is it possible to use connection string to connect DB? if yes I think better to do it like that so we have consistent way to connect DB for all of our DB extractors

this is related with https://github.com/odpf/meteor/pull/290

Reply:
Sure, We can add the connection to DB via a connection string.